examples/llm/benchmarks/plot_pareto.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import re
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from matplotlib.ticker import MultipleLocator
+
+
+def get_genai_perf_profile_export_json_paths(search_paths):
+    paths = []
+    for search_path in search_paths:
+        for root, dirs, files in os.walk(search_path):
+            for file in files:
+                if file == "profile_export_genai_perf.json":
+                    paths.append(os.path.join(root, file))
+    return paths
+
+
+# search for -concurrency<number> in the name
+def parse_concurrency(name):
+    matches = re.findall(r"-concurrency(\d+)", name)
+    if len(matches) != 1:
+        raise Exception(f"non-unique matches: {matches}")
+    concurrency = 0
+    for c in matches:
+        concurrency += int(c)
+    return concurrency
+
+
+# serach for _tp<number>dp<number> in the name
+def parse_gpus(name):
+    matches = re.findall(r"_tp(\d+)dp(\d+)", name)
+    if len(matches) <= 0 or len(matches) > 2:
+        raise Exception(f"invalid matches: {matches}")
+    total_gpus = 0
+    for tp, dp in matches:
+        total_gpus += int(tp) * int(dp)
+    return total_gpus
+
+
+def extract_val_and_concurrency(genai_perf_profile_export_json_paths, stat_value="avg"):
+    results = []
+    for json_path in genai_perf_profile_export_json_paths:
+        with open(json_path, "r") as f:
+            data = json.load(f)
+            # output_token_throughput contains only avg
+            output_token_throughput = data.get("output_token_throughput", {}).get("avg")
+            output_token_throughput_per_request = data.get(
+                "output_token_throughput_per_request", {}
```

Comment:
Have you tested this in latest containers using genai-perf >= 0.0.13? I think this field was renamed to `output_token_throughput_per_user`. It should be a simple find/replace to update I think.